tests/serializers.py

```diff
@@ -170,6 +171,10 @@ class Meta:
         requires=['location.cat_set.*'],
         deferred=True
     )
+    number_of_katz = DynamicMethodField(
+        requires=['location__cat_set'],
```

Comment:
Yeah, so what happened with the recent change is that the `location` and `cat_set` are now separated, so we build a Prefetch on Location which then prefetches `cat_set`. Before, it was basically `User.object.prefetch_related('location__catset')` so there was no separate explicit prefetch on Location. [This test](https://github.com/AltSchool/dynamic-rest/pull/87/files#diff-625efbbf35a2290861f596fee1b19838R934) shows how these two fields have different behaviors.